packages/babel-plugin-import-jsx-pragma/package.json

```diff
@@ -24,7 +24,7 @@
 		"url": "https://github.com/WordPress/gutenberg/issues"
 	},
 	"engines": {
-		"node": ">=12"
+		"node": ">=14"
```

Comment:
Should we also bump it for the `babel-plugin-makepot` plugin? Or `library-export-default-webpack-plugin`?

Reply:
I skipped them as they aren't actively used in the project.

I belive `babel-plugin-makepot` might get an update as it could be used by some external projects.

For the webpack plugin, we should deprecate it instead, as it is now natively supported in webpack 5. I think there is another webpack plugin that we no longer use since we migrated to webpack 5 due to the same reasons.